src/platforms/unity/native-support/index.mdx

```diff
@@ -4,15 +4,21 @@ description: "Learn how the Unity SDK handles native support."
 sidebar_order: 100
 ---
 
+Currently, the Unity SDK offers native support for `Android`, `iOS`, and `Windows`. The support is `enabled` by default and you can opt out for each platform individually in the configuration window.
+
+## Android & iOS
+
 <Note>
 
-The setup for the native SDKs happens during build time with the options defined in the Sentry editor configuration window. Changes to the options object done through [ScriptableOptionsConfiguration](/platforms/unity/configuration/options/) will not affect events from the native layer.
+The setup for the iOS and Android native SDKs happens during build time with the options defined in the Sentry editor configuration window. Changes to the options object done through [ScriptableOptionsConfiguration](/platforms/unity/configuration/options/) will not affect events from the native layer.
 
 </Note>
 
-Native support is `enabled` by default. The Unity SDK passes the options defined through the Sentry Unity Editor window during build time. For Android, it embeds the [Android SDK](/platforms/android/) inside the Gradle project created by Unity. The [iOS SDK](/platforms/apple/guides/ios/) is embedded inside the generated Xcode project. This means there's no need to use the Unity [built-in crash reporting](https://docs.unity3d.com/ScriptReference/CrashReport.html) functionality.
+The Unity SDK passes the options defined through the Sentry Unity Editor window during build time. For Android, it embeds the [Android SDK](/platforms/android/) inside the Gradle project created by Unity. The [iOS SDK](/platforms/apple/guides/ios/) is embedded inside the generated Xcode project. This means there's no need to use the Unity [built-in crash reporting](https://docs.unity3d.com/ScriptReference/CrashReport.html) functionality.
+
+## Windows
 
-You can opt out of native support by unchecking `iOS Native Support` and `Android Native Support` in the configuration window.
+The Unity SDK initialized the Native SDK through the C# layer, and this means that we currently do not capture errors of the Unity engine itself.
```

Comment:
I wonder if we can clarify a bit how it works (and its straights) before we point out this "Engine crashes" situation which most ppl don't come here expecting (it's kind of a unique feature we added on Android and iOS).

```suggestion
Windows native crashes are automatically captured through minidumps. Minidump capture is configured and enabled once the C# scripting layer runs, as early as possibly after Unity engine starts. This is slightly different than the Android and iOS support where we inject native initialization code that runs before Unity engine runs.
```